charts/gitops-runtime/templates/_components/internal-router/_deployment.yaml

```diff
@@ -38,20 +38,20 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           ports:
             - name: http
-              containerPort: 80
+              containerPort: 8080
               protocol: TCP
           readinessProbe:
             initialDelaySeconds: 10
             timeoutSeconds: 10
             httpGet:
-              port: 80
+              port: http
```

Comment:
Shouldn't this and the livenessProbe be also 8080?

Reply:
I think we also need to change the `targetPort` in the internal-router service to 8080 @ilia-medvedev-codefresh

Reply:
yeah, i had to make these two changes in csdp-official in order to get the cli-runtime running correctly